core/src/main/java/hudson/tasks/Fingerprinter.java

```diff
@@ -92,13 +94,36 @@
      */
     private final String targets;
 
+    /**
+     * Default null 'excludes' pattern as in Ant.
+     */
+    private String excludes = null;
+
+    /**
+     * Default ant exclusion
+     */
+    private boolean defaultExcludes = true;
+
+    /**
+     * Indicate whether include and exclude patterns should be considered as case sensitive
+     */
+    private boolean caseSensitive = true;
+
     @Deprecated
     Boolean recordBuildArtifacts;
 
     @DataBoundConstructor public Fingerprinter(String targets) {
         this.targets = targets;
     }
 
+    // @DataBoundConstructor
```

Comment:
Need feedback on how this annotation is used. I can't figure out what it does.

Reply:
There is already a `@DataBoundConstructor`; you may not have two.

You _may_ delete the new constructor and instead define three new `@DataBoundSetter`s. In that case you _may_ also extend `Fingerprinter/config.jelly` to allow these things to be configured from the standalone fingerprint step (not merely from its indirect usage in archive artifacts).